packages/prism-theme-odyssey/src/theme.scss

```diff
@@ -0,0 +1,154 @@
+:root {
+  --font-size: 1rem;
```

Comment:
Shouldn't all of this be sourced from ODS?

Reply:
Fixed.